From 885aabdb313685405737112dcb7d7774eee6d879 Mon Sep 17 00:00:00 2001
From: Thomas Berg <merlin66b@gmail.com>
Date: Thu, 19 Oct 2017 14:02:21 -0500
Subject: [PATCH] SConscript: Support python2 print statements

This fixes a regression introduced in scons-3.0.0, where
SConscripts containing python 2 print statements would cause
syntax errors even when executing scons with python 2.7.

This ensures backward compatibility, allowing users to build
legacy code with scons-3.0.0 without having to patch it.

Taken from
https://github.com/SConsProject/scons/commit/2e0de3c55f22b3eaa7767b69740b898f3d2f46bf

Upstream-Status: Accepted

Signed-off-by: Jose Lamego <jose.a.lamego@linux.intel.com>
---
 engine/SCons/Script/SConscript.py | 2 --
 1 file changed, 2 deletions(-)

diff --git a/engine/SCons/Script/SConscript.py b/engine/SCons/Script/SConscript.py
index bc05540..332d1fa 100644
--- a/engine/SCons/Script/SConscript.py
+++ b/engine/SCons/Script/SConscript.py
@@ -5,8 +5,6 @@ files.
 
 """
 
-from __future__ import print_function
-
 #
 # Copyright (c) 2001 - 2017 The SCons Foundation
 #
-- 
2.7.4

